src/Microsoft.AspNetCore.AzureLightup/AzureStartupLoader.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Hosting;
+
+[assembly: HostingStartup(typeof(Microsoft.AspNetCore.AzureLightup.AzureStartupLoader))]
+
+namespace Microsoft.AspNetCore.AzureLightup
+{
+    /// <summary>
+    /// A dynamic azure lightup experiance
+    /// </summary>
+    public class AzureStartupLoader : IHostingStartup
+    {
+        /// <summary>
+        /// Calls UseAzureAppServices
+        /// </summary>
+        /// <param name="builder"></param>
+        public void Configure(IWebHostBuilder builder)
+        {
+            builder.UseAzureAppServices();
```

Comment:
No, the dynamic light up. I want to make sure we test this with 2 versions before we ship. You can push this change for now so that we can test it but I'd like to see the code we need to use once we are reflection based.